src/locals/hu.json

```diff
@@ -0,0 +1,145 @@
+    "minimization_of_the_positive": "A Sikerek elhallgatása",
+    "minimization_of_the_positive_explanation": "Ha lebecsüljük a sikereinket, akkor minimalizáljuk a (dopamint-tüzelést) [ez hogy mondjuk magyarul? ¿Öröm?]. Akkor is, ha az első napunk nem ment 100%-ig a tervnek megfelelően, nem jelenti azt, hogy pl. 60% ami jól ment azt ignorálhatjuk.",
+    "minimization_of_the_positive_thought": "Sok embernek tetszett az előadásom, de elbaltáztam a bevezetést, szóval biztos elcsettintettem.",
+    "other_blaming": "Mások hibáztatása",
+    "other_blaming_explanation_1": "Ha egy rossz helyzetet mindig valaki másnak a hibájának gondolunk, akkor másokat hibáztatunk. Ha megbuktál egy vizsgán és a tanárt hibáztatod, akkor rossz irányba feccölöd a megoldásra szánt energiáidat. Valaki eléd vágott, az autópályán? Ha rádudálsz, rátapadsz, az segít? Most közveszélyes és ideges is vagy egyszerre!",
+    "other_blaming_explanation_2": "Ez nem jelenti azt, hogy minden negatívumért csak magadat kell okolnod. Nem kell senkit sem hibáztatni. Senkinek sem kell hibáznia ahhoz, hogy hiba történjen. Engedd el, hibáztatás nélkül.",
+    "other_blaming_thought": "Ez a genya előttem túl sokat vacakol a sorban!",
+    "over_generalization": "Túl-álltalánosítás",
+    "over_generalization_explanation": "Ha csak egy példa alapján messzemenő következtetéseket vonunk le, akkor túl-állatlánosítunk. Ha elbaltáztad a prezidet és emiatt azt gondolod, hogy \"rossz\" vagy az előadásban, akkor te túl-álltalánosítasz.",
+    "over_generalization_thought": "Senki sem kért-fel táncolni, szal' biztos nem is fog.",
+    "self_blaming": "Önvád",
+    "self_blaming_explanation": "Ha csak magadat okolod egy kialakult szituációért, az az önvád. Nem kell minden rossz dologért felelősnek lenned. Ha a dugóban ragadsz és azon őrlődsz, hogy miért nem indultál el korábban, az önvád. Mással is így bánnál ebben a szituációban?",
+    "self_blaming_thought": "A fiam rosszalkodik az iskolában, biztos rosszul nevelem őt.",
+    "should_statements": "Hibás Biztos állítások",
+    "should_statements_explanation_1": "Ha valakiket felruházol olyan tulajdonságokkal, amikkel valójában nem rendelkeznek, akkor te helytelenül használod a \"biztos\" szót. Ha pl. félsz a repüléstől és a belső monológod a \"Nem kéne ettől félnem, biztos nincs semmi baja a gépnek!\" akkor csak feleslegesen terheled magad. Repülési fóbiád van! Repülési fóbiában szenvedőknél normális, ha félnek a repüléstől!",
+    "should_statements_explanation_2": 
+    "A hibás biztos állítások nagyon hülyén vehetik ki magukat, amikor hangosan kimondod; ez a lényeg! Nem illogikusak!",
+    "should_statements_thought": "Felnőtt ember vagyok, nem kéne így hisztiznem.",
+    "accessibility": {
+        "help_button": "Súgó",
+        "list_button": "Gondolatok listázása",
+        "new_thought_button": "Új gondolat",
+        "settings_button": "Beállítások",
+        "delete_thought_button": "Gondolat elfelejtése"
```

Comment:
... törlése?

Reply:
I would like to use the "elfelejtése" metaphor to enhance ease of use of the application.